CONTRIBUTING.md

```diff
@@ -175,10 +175,67 @@ the page to activate debug mode. The state of debug mode will be remembered by
 `window.sessionStorage` and will not survive a browser restart or opening the
 docs in a new tab.
 
-### Invisible index anchors
-
-You can create index entries and invisible anchors with `X<|thing,category>`.
-To make them visible activate debug mode.
+### Search indexing rules
+
+Bits of the content can be indexed for quick navigation.
+
+An index item is created using the Pod formatting code `X` in those formats:
```

Comment:
Also an anchor. Is this explained somewhere else in the document?

Reply:
You mean a HTML anchor? I believe it's an implementation detail, no?

Reply:
One people contributing to the documentation needs to be aware of. That's why indexing is placed on top of sections, for instance, not below them.